modules/hosting/pantheon_hosting/pantheon_hosting.info

```diff
@@ -1,5 +1,5 @@
-name = Pantheon Core
+name = Pantheon Hosting
```

Comment:
I guess this module isn't needed at all currently. Should just do nothing if the environment is Pantheon.

Reply:
We are enabling Pantheon specific modules... just not as dependencies.  We also need https://github.com/CuBoulder/express/issues/1784